Makefile

```diff
@@ -113,6 +113,18 @@ srpm: .srpm-done
 
 rpm: .rpm-done
 
+.generic-rpm-done:
```

Comment:
should this also depend on the sources.tgz target like .rpm-done does?

Reply:
sources.tgz is only copying files under AL folder https://github.com/aws/amazon-ecs-init/blob/master/Makefile#L87-L92. so i added some similar source preparation commands in the .generic-rpm-done target: https://github.com/aws/amazon-ecs-init/pull/389/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R118-R120